### PR TITLE
GameState Editor & Viewer

### PR DESCRIPTION
**Pull request by Zychronix**

### CODE REVIEW

File: src/main/java/legend/game/debugger/GameStateEditorController.java [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+  public TextField textDotOffset;
+  @FXML
+  public TextField textFacing;
+  @FXML
+  public TextField textAreaIndex;
+
+  public void initialize() {
+    this.textData1.setText(String.valueOf(gameState_800babc8._04));
+
+    for (int i = 0; i < 0x20; i++) {
```
Comment (LordMonoxide) [inline src/main/java/legend/game/debugger/GameStateEditorController.java:106]: CS: no space after `for`

Comment (LordMonoxide) [reply src/main/java/legend/game/debugger/GameStateEditorController.java:106]: Everywhere

Comment (LordMonoxide) [reply src/main/java/legend/game/debugger/GameStateEditorController.java:106]: Also ifs :P

**Review by LordMonoxide — CHANGES_REQUESTED** [review]

File: src/main/java/legend/game/debugger/GameStateEditorController.java (hunk 1 above)
Comment (LordMonoxide) [reply src/main/java/legend/game/debugger/GameStateEditorController.java:106]: Also the other files

Comment (Zychronix) [reply src/main/java/legend/game/debugger/GameStateEditorController.java]: Fixed in the latest commit

**Review by LordMonoxide — APPROVED** [review]

**Pull request merged**